sql/core/src/main/scala/org/apache/spark/sql/execution/Columnar.scala

```diff
@@ -538,7 +539,13 @@ case class ApplyColumnarRulesAndInsertTransitions(
     } else if (plan.supportsColumnar) {
       // `outputsColumnar` is false but the plan outputs columnar format, so add a
       // to-row transition here.
```

Comment:
kinda outdated comment

Reply:
It's still meaningful, just needing to move it.